### DISCUSSION TITLE
Add a Offset for MHZ19 Sensors

### DISCUSSION THREAD

**Discussion by HoerMirAuf** [arendst/Tasmota#15048] [Ideas]
Hi ....
is there a possibilty to add a co2offset command? Similar the tempoffset?

I'm using MHZ19**B** and MHZ19**E** Sensors. The MHZ19**E** meassure is quite linear to the "B" Version only some PPM (round about 100-120) higher.

It would be nice to adjust the meassure output to the meassure Range of the B Sensor.

**Comment by Jason2866** [comment]
Have you calibrated the CO2 sensor?

**Comment by HoerMirAuf** [comment]
Yes of course ... all Sensors have been calibrated to 400PPM.
